.github/workflows/storage-server-images.yml

```diff
@@ -51,7 +51,9 @@ jobs:
         with:
```

Comment:
NIT (line 12, `IMAGE_NAME`): maybe we can make this workflow similar to `publish-server` and just use `tursodatabase/libsql-storage-server-devel` without `github.repository` interpolation?

Reply:
we could, but then I won't be able to run trigger this workflow in my fork. May be once it is production ready and set,  we can rename the `IMAGE_NAME`. WDYT?

Reply:
Ah, I see. This is NIT - so we can skip this for sure

But why are you developing this in fork actually?)

Reply:
I use my fork as a testbed. Easier to test changes. For e.g. Github workflows don't run properly unless they are in `main`. So to test, I merge in my fork's main, test, modify etc. Another reason is just habit, to make changes in my fork and send PR from them.